misc/specs/atrium-specs-common/src/main/kotlin/ch/tutteli/atrium/specs/integration/AnyAssertionsSpec.kt

```diff
@@ -40,6 +40,7 @@ abstract class AnyAssertionsSpec(
     isNotInDataClass: Fun1<DataClass, Iterable<DataClass>>,
     isNotInNullableInt: Fun1<Int?, Iterable<Int?>>,
     isNotInNullableDataClass: Fun1<DataClass?, Iterable<DataClass?>>,
+    because: Fun2<String, String, Expect<String>.() -> Unit>,
```

Comment:
This is unused momentarily, therefore one build failed.
I am not sure why we need it and how to use it exactly in the spec context.

Reply:
This parameter is the function that should be tested. Your test currently tests the `because` implementation from the `fluent` API. However, this test class is meant to be generic, meaning that it can test both the `infix` and the `fluent` API. 

The test cases you write should never call `because` from the `fluent` API. Instead, they should use this parameter. Atrium has some helpers to achieve this. If you look at the other test cases, you can find out how to use them.

If you need more help, do not hesitate to ask!